File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        String odd_str = "";
        String even_str = "";
        
        for(int i=0; i<num_list.length; i++){
            if((num_list[i]%2)==0){
                even_str += String.valueOf(num_list[i]);        // 짝수를 이어 붙임
            }
            else{
                odd_str += String.valueOf(num_list[i]);         // 홀수를 이어 붙임
            }               
        }
        
        answer += (Integer.valueOf(odd_str) + Integer.valueOf(even_str));
        
        return answer;
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    var answer = 0;
    var odd_str = "";
    var even_str = "";
    
    for(i of num_list){
        if(i%2==0){ // i가 짝수인 경우
            odd_str += i.toString();    // i를 문자열로 변환
        }
        else{       // i가 홀수인 경우
            even_str += i.toString();
        }
    }
    
   answer = Number(odd_str) + Number(even_str); // i를 숫자로 변환
    
    return answer;
}